### PR TITLE
Update Frontity packages and other minor chores

### PR DESCRIPTION
**Pull request by luisherranz**
I was updating the Frontity packages but for some reason, I found the following error: 

Screenshot 2021-07-01 at 17 04 01

It doesn't seem related to this PR, because the same happens in `master`.

Other than that I have: 
- Created a `Packages` type export.
- Added a `tsconfig.json`.
- Renamed the Vercel config file.
- Refactored some types.

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit d39b65761bb89b1f58a9c7fceba485991296db45

https://frontity-org-7gswz10u0-frontity.vercel.app

**Comment by luisherranz** [comment]
It doesn't happen in production: https://frontity-org-7gswz10u0-frontity.vercel.app/

Any idea why this is happening? Has anyone seen this problem before?

**Comment by DAreRodz** [comment]
I saw the same error before, but it was supposed to be fixed in this commit: https://github.com/frontity/frontity.org/commit/f5dcdb31a253b801113e9e10b5ce04d007be2ca2

**Comment by DAreRodz** [comment]
It was supposed to be fixed in https://github.com/frontity/frontity.org/pull/180, actually. That time it was related to npm 7 installing `peerDependencies`.

**Comment by luisherranz** [comment]
I have run `npm install` with npm 6 but I'm still getting the same problem.

**Comment by DAreRodz** [comment]
> I have run `npm install` with npm 6 but I'm still getting the same problem.

You could try running `npm install --legacy-peer-deps` with npm 7 and see if it works that way. The weird thing is that the packages adding the conflicting peerDeps were already removed in #180 so it could be a different issue this time.

**Comment by luisherranz** [comment]
Wait, I had a `node_modules` folder inside `packages/frontity-org`...

**Comment by luisherranz** [comment]
Ok, now it works. So the problem was that nested `node_modules` folder containing a `react` install.

I'm going to go back to npm 7 and start over to see if the problem perists.

**Comment by luisherranz** [comment]
Ok, it works fine with npm 7 as well. So it seems like it was totally my fault due to the nested `node_modules` folder 🙂

Ready for review then!

File: .github/workflows/deploy-and-lighthouse.yml [hunk 1]
```diff
@@ -26,8 +26,8 @@ jobs:
         env:
           COMMIT_SHA: ${{ github.sha }}
 
-      - name: Deploy to now
-        run: npx now --confirm --token $NOW_TOKEN --meta commit=${GITHUB_SHA} --meta branch=${GITHUB_REF} --scope frontity | tee >(grep '^https:\/\/frontity-org-' > deploy-url.txt)
+      - name: Deploy to vercel
+        run: npx vercel --confirm --token $NOW_TOKEN --meta commit=${GITHUB_SHA} --meta branch=${GITHUB_REF} --scope frontity | tee >(grep '^https:\/\/frontity-org-' > deploy-url.txt)
         env:
           NOW_TOKEN: ${{ secrets.NOW_TOKEN }}
```
Comment (DAreRodz) [inline .github/workflows/deploy-and-lighthouse.yml:32]: Would it make sense to change also this env variable to something like `VERCEL_TOKEN`?

**Review by DAreRodz — APPROVED** [review]
Everything OK, @luisherranz :+1:

File: .github/workflows/deploy-and-lighthouse.yml (hunk 1 above)
Comment (luisherranz) [reply .github/workflows/deploy-and-lighthouse.yml:32]: I don't know how to rename a secret, so I'm just going to leave that for now.

**Pull request merged**